packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -281,6 +282,9 @@ class CupertinoDatePicker extends StatefulWidget {
   /// null.
   final ValueChanged<DateTime> onDateTimeChanged;
 
+  /// Background color behind the children.
```

Comment:
nit: `CupertinoDatePicker` doesn't have children so referring to "children" here is a bit confusing. Also by convention docs should mention what the default color is and the nullability of this field. e.g.:

https://github.com/flutter/flutter/blob/bd3305b14bf189ceca10a9326013cf30d102bcfa/packages/flutter/lib/src/cupertino/switch.dart#L97-L101

Reply:
added suggested comments